<?xml version="1.0"?>
<launch>
	
	<node name="itst2data_pub" pkg="ref_data_manager" type="itst2data_pub" output="screen" >
		<rosparam file="$(find ref_data_manager)/config/pr_info.yaml"/>

		<param name="Number_of_candidate" value="5" />
		<param name="IMAGE_FLAG" value="false" />

		<remap from="/score/vis" to="/score/vis/itst" />
		<remap from="/score/best" to="/score/best/itst" />
	</node>
	
</launch>


